src/main/java/core/basesyntax/ToDrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface ToDrawFigure {
+    String toDraw();
```

Comment:
```suggestion
   void draw();
```

за умовою завдання цей метро має бути void, і всередині викликати System.out.println()